src/platform/qt/library/LibraryController.h

```diff
@@ -47,16 +53,18 @@ struct LibraryEntry {
 	size_t filesize;
 	uint32_t crc32;
 
+	LibraryEntry& operator=(const LibraryEntry&) = default;
+	LibraryEntry& operator=(LibraryEntry&&) = default;
 	bool operator==(const LibraryEntry& other) const { return other.fullpath == fullpath; }
+
+	static inline uint64_t checkHash(size_t filesize, uint32_t crc32) {
```

Comment:
You might want to add a const mLibraryEntry* overload for this and make the one that takes the specific types private so it can be changed easily in the future, should we decide the current inputs aren't enough.

Reply:
Tunnel vision on the feature, not the fine details, I would wager.